### ISSUE TITLE
Multiple Product Category for each sku

### ISSUE THREAD

**Issue by singchung** [dereklokgithub/PSM-UAT#12]
1. Should we put XD design products into corporate gift Cat? 
   I think "corporate gift" is products by category
   "branded premium" actually is products by brand (suggest we can provide a branding such as PSM brand for OEM items)

If agree, we need to work together for refine all products to multi-cat.

**Comment by dereklokgithub** [comment]
@singchung Agreed. We can support multiple category. Suggest to do it in the next phase as 1. we need to re-define all the root category and sub-categories 2. then we need to apply multiple categories to each sku and do the product upload again.

**Comment by dereklokgithub** [comment]
Done. See below example

![image](https://cloud.githubusercontent.com/assets/13758674/16614869/22d18dfc-43a6-11e6-905b-d691a0b392f8.png)
![image](https://cloud.githubusercontent.com/assets/13758674/16614979/4b81a1f6-43a6-11e6-97a1-b4bbcb0c77cd.png)
